### ISSUE TITLE
Action Facade\Ignition\Http\Controllers\ExecuteSolutionController not defined.

### ISSUE THREAD

**Issue by realtebo** [facade/ignition#227]
This is my composer.json
```

    "require": {
        "php": "^7.2",
        "doctrine/dbal": "^2.10",
        "fideloper/proxy": "^4.0",
        "guzzlehttp/guzzle": "~6.0",
        "laravel/framework": "^6.0",
        "laravel/tinker": "^1.0",
        "shalvah/laravel-jsend": "^1.2"
    },
    "require-dev": {
        "facade/ignition": "^1.4",
        "fzaninotto/faker": "^1.4",
        "laravel/ui": "^1.1",
        "mockery/mockery": "^1.0",
        "nunomaduro/collision": "^3.0",
        "phpunit/phpunit": "^8.0"
    },
```

I cannot understand what causes this error, and I cannot diagnose

![image](https://user-images.githubusercontent.com/1268699/70731308-c88e4e00-1d06-11ea-832b-3bb9ad2df4b3.png)

**Comment by realtebo** [comment]
Found:

2 routes with same action and uri but different name causes this when calling the 2nd defined route

**Comment by realtebo** [comment]
Found 

This causes the problem

```
return view('app.user.home', [
            'intent' => $request->get('intent'),
]);

```
but this not

```
return view('app.user.home', [
            'inteniont' => $request->get('intent'),
]);

```

Why a variable name passed to a viewn causes an error in the ignition package?

**Comment by realtebo** [comment]
I removed my previous comment because was not directly the cause.

I discoverred that: to trigger this warning, simply use in a view an undefined variabile.

**Comment by wbamert** [comment]
I'm writing to concur on this error. The new error pages using the ignition system fail when a variable fails. i can reproduce this on my site rather easily. The real issue on the page was a divide by zero error but the error page errors out and that's all you see.

![2019-12-16 13_01_01-Action Facade not defined](https://user-images.githubusercontent.com/46751652/70935183-3c7d6e80-2005-11ea-9d03-361099751ec0.png)
![2019-12-16 13_01_01-Division by zero (View_ _var_www_datadev motorwerks com_htdocs_resources_view](https://user-images.githubusercontent.com/46751652/70935184-3c7d6e80-2005-11ea-9d2e-c600f1bbb72a.png)
![real_error](https://user-images.githubusercontent.com/46751652/70935185-3c7d6e80-2005-11ea-9288-d0f9032a9ecb.PNG)

Any help would be appreciated.

Walter

**Comment by hareendranmg** [comment]
Any updates? Same issue here for a fresh laravel project.

**Comment by lashto** [comment]
Had same error on linux. Giving write access to storage /solved the problem

**Comment by UnderlineWords** [comment]
+1 same issue here.

**Comment by MajedDH** [comment]
As a workaround I had to downgrade from ^1.4 (which was resolved to 1.14) to exactly 1.4  in my composer.json . now I'm waiting for this to be fixed !

**Comment by hmhrex** [comment]
I'm having this issue as well. In my case it's when I try to convert a collection into a json object. It fails when using any of the following:
`response()->json($collection);`
`json_encode($collection);`
`$collection->toJson();`

**Comment by peterchrjoergensen** [comment]
I'm also having this issue on Laravel v.6.12.0 and PHP 7.2.

**Comment by aglipanci** [comment]
+1 happening to me as well.

**Comment by donovanhare** [comment]
This happened to me when I had invalid code in my `AppServiceProvider.php`, after removing the rogue code the error disappeared.

**Comment by nqxcode** [comment]
This problem is similar to https://github.com/facade/ignition/pull/204.
We must solve it in the same way.

**Comment by khairahmanplus** [comment]
same issue here, triggered when you have undefined variable.

**Comment by realtebo** [comment]
It's very annoying. We're actively working on a few projects all using Laravel 6 and this bug actively disrupt our day-by-day coding activity. Is there no plan to resolve? 

Does this happens on Laravel 7?

**Comment by tom-it** [comment]
I had this error when a git conflict occured inside a serviceprovider. after resolving the error it was all back to normal.

**Comment by fixwa** [comment]
Same here. After adding a simple call to a method in an existing repository, I receive this error.

**Comment by freekmurze** [comment]
This issue has been fixed by #265.

**Comment by realtebo** [comment]
This fix is really appreciated. But it simply hide the error. I do not understand probably, but I think it is not resolving the root cause, that is actually unknown
